WordPress/Classes/ViewRelated/Media/MediaProgressCoordinatorNoticeViewModel.swift

```diff
@@ -11,6 +12,9 @@ struct MediaProgressCoordinatorNoticeViewModel {
 
         self.mediaProgressCoordinator = mediaProgressCoordinator
         self.progress = progress
+
+        let failedMediaIDs = mediaProgressCoordinator.failedMediaIDs
```

Comment:
Maybe this two lines should be refactored to a method inside the mediaProgressCoordinator?